app/pods/search-schema/create/route.js

```diff
@@ -0,0 +1,49 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+    model(params) {
+        return this.explorer.getCluster(params.clusterId, this.store);
+    },
+
+    actions: {
+        createSchema: function(clusterId, schemaName, schemaContent) {
+            let self = this;
+            let xmlDoc = null;
+            let url = `/riak/clusters/${clusterId}/search/schema/${schemaName}`;
+
+            try {
+                xmlDoc = Ember.$.parseXML(schemaContent);
+            } catch(error) {
+                // TODO: Put in proper error messaging
+                alert('Invalid XML. Please check and make sure schema is valid xml.');
+                return;
+            }
+
+            if (!Ember.$(xmlDoc).find('schema').attr('name')) {
+                // TODO: Put in proper error messaging
+                alert('Solr requires that the schema tag has a name attribute. Please update your xml.');
+                return;
+            }
+
+            if (!Ember.$(xmlDoc).find('schema').attr('version')) {
+                // TODO: Put in proper error messaging
+                alert('Solr requires that the schema tag has a version attribute. Please update your xml.');
+                return;
+            }
+
+            return Ember.$.ajax({
+                type: 'PUT',
+                url: url,
+                contentType: 'application/xml',
+                processData: false,
+                data: xmlDoc
+            }).then(function(data) {
+                alert('schema created!');
```

Comment:
While I was able to get a `204` response from the server, I don't know how to associate this schema to a an index. Right now through our UI, the only way to get to schemas is through ones that have been associated to indexes (as that is how they are created through Ember Data. reference the updates in `services/explorer.js`). I could create a new schema in ED right here and then transition to that route, but once you leave there would be no way to get back to that view without manually typing in the url.

Any thoughts on how this should be handled?